code/modules/client/preference/loadout/loadout_racial.dm

```diff
@@ -0,0 +1,40 @@
+/datum/gear/racial
+	sort_category = "Racial"
+	subtype_path = /datum/gear/racial
+	cost = 2
+
+
+/datum/gear/racial/tajciv
+	display_name = "embroidered veil"
+	description = "a common traditional nano-fiber veil worn by many Tajaran, It is rare and offensive to see it on other races."
```

Comment:
Not gonna copy-paste this on each case, but all of these datums' desc begin with an un-capitalized "a". Might want to capitalize that.